docs/en_US/hpo_benchmark.rst

```diff
@@ -8,11 +8,11 @@ Terminology
 ^^^^^^^^^^^
```

Comment:
as this doc is introducing hpo benchmarks, it is better to first list all the benchmarks that we have supported in a table. then introduce how to use each benchmark